app/helpers/request_helper.py

```diff
@@ -11,6 +16,10 @@ def service_name(url=None):
             return 'SDX_STORE'
         elif 'sequence' in parts:
             return 'SDX_SEQUENCE'
+        elif 'cora' in parts:
+            return 'SDX_TRANSFORM_CORA'
+        else:
```

Comment:
John questioned "What happens if none of these match?" so thought it might be better to have an explicit return.